include/cell/copy/global_to_shared.hpp

```diff
@@ -229,17 +223,16 @@ struct SharedToGlobalStorerImpl<Shared_, Global_, kRowExec_, kColExec_,
     static constexpr int kColExec = kColExec_;
 
     // strides to iterate over each 16x16 `BaseTile` in the shared memory
-    static constexpr int kSrcCstride = BaseShape::kCols * Shared::kColStride;
-    static constexpr int kDstCstride = BaseShape::kCols * Global::kColStride;
-
-    static constexpr int kRstride = BaseShape::kRows;
+    // static constexpr int kSrcCstride = BaseShape::kCols * Shared::kColStride;
```

Comment:
Does this comment still serve a purpose?

Reply:
deleted.